Scripts/script-StixParser.yml

```diff
@@ -67,6 +67,7 @@ type: python
 tags:
 - stix
 - ioc
+comment: "-"
```

Comment:
@asafshen

Reply:
@meirwah can you help me with this comment please?

Reply:
'Parse Stix files to Demisto indicators '